documentation/projects/proposals/additional_search_views/20230424-project_proposal_additional_search_views.md

```diff
@@ -0,0 +1,36 @@
+# 2023-04-24 Project Proposal
+
+## Reviewers
+
+- [ ] @obulat
+- [ ] @krysal
+
+## Project summary
+
+The Openverse team has decided to enhance the browsing experience by
+incorporating new content views for providers, tags, and creators. This feature
+will allow users to share collections detached of individual searches.
+
+## Goals
+
+Search Experience
+
+## Requirements
+
+## Success
+
+The project will be considered successful when each tag, provider, and creator
+has a standalone and shareable page
```

Comment:
I think we should add analytics here, too:
Something like "And we have the analytics events set up to measure how users are using them"

Reply:
I'm not sure about this. We don't have a criterion to mark its utility in relation with other search flows or as a standalone browsing experience. We surely need to assess how these pages are used, but as inputs for understanding how people search and browse and to polish the tool rather than labeling this project as success or not.

Reply:
By "adding analytics" I meant adding the code that will send an analytics event when the user opens one of the additional search views. So, when a user opens a provider page, we send an event to the Analytics server with the provider name and other relevant details. Without this event, we would not be able to see if the users are actually opening the page or not. We can, of course, add the events after this project is finished. But I think it would be useful to add the analytics events to any important frontend user-facing changes. Just like we add tests for any visual changes.

This **does not** include the actual collection and analysis of this data. I wasn't suggesting that we decide whether this project is successful by the number of users who opened the views.

Does this comment make my point clearer, @panchovm ? Or do you think that we should not add any code that sends analytics events as part of this project?

Reply:
Analytics tracks pageviews by default, so at the very least we should be getting information about how many visits these pages are receiving.

Reply:
I think it would be useful to have a separate Plausible event, with properties like {"searchView" : <"tag"|"provider"|"creator">, "detail": <"tagName"|"providerName"|"creatorUrl/creatorName/creatorProvider">} to track this.

Reply:
FWIW custom events cost money (each one counts towards our "pageviews" for the billable period), so if we can derive the information from an existing one, it's worth doing so. If we truly need another event, of course that is fine, but we should keep in mind the cost and avoid duplicating or denormalising event data.

Reply:
Awesome. I will state in the document: Track the uses of these new pages by derivating data from existing events.